src/style.css

```diff
@@ -1 +1,81 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --blue: #00acdc;
+}
+
+.header {
+  position: relative;
+  /* 12px */
+  font-size: 0.75rem;
+  /* 60px */
+  height: 3.75rem;
+}
+
+.logo {
+  height: 40px;
+  /* center vertically */
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  left: 50px;
+}
+
+.nav {
+  font-family: Roboto, sans-serif;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: flex-end;
+  list-style-type: none;
+}
+
+.nav__item {
+  margin: 0 10px;
+}
+
+.nav__item:first-of-type {
+  margin-left: 0;
+}
+
+.nav__item:last-of-type {
+  margin-right: 0;
+}
```

Comment:
if you're consistent applying only one-end margin you could save few lines of code.
Say, keep only right margin - then just cancel it for `last-child / of-type` 
- opposed to cancelling margins for both first and last